### ISSUE TITLE
Using the wrapper hangs the app (it does nothing)

### ISSUE THREAD

**Issue by TheFizz** [MingweiSamuel/Camille#2]
Hit me if I don't get smth correctly, but i can't get Camille to work.

```
string summonerName = "thefizz";
MingweiSamuel.Camille.Enums.Region RU = MingweiSamuel.Camille.Enums.Region.RU;
var summoner = api.Summoner.GetBySummonerName(RU, summonerName);
```

When I start debugging the app simply does nothing. It doesn't even throw an exception. Maybe I'm missing something?

**Comment by MingweiSamuel** [comment]
Hi, I am unable to reproduce the issue. Could you gist your code and also say what version of Camille and .NET you are using?

**Comment by TheFizz** [comment]
Hello! Thanks for the response!

Just created the sipliest i can imagine.
IDK if it's of some use, but when `using MingweiSamuel;`  VS2017 says `Using directive is unnecessary `
```
using System.Windows.Forms;

namespace CamilleTest
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            var api = MingweiSamuel.Camille.RiotApi.NewInstance("RGAPI-91e2bf25-f491-4e56-b8fb-08c6f66d0b8b");
            string summonerName = "thefizz";
            MingweiSamuel.Camille.Enums.Region RU = MingweiSamuel.Camille.Enums.Region.RU;
            var summoner = api.Summoner.GetBySummonerName(RU, summonerName);
            label1.Text = summoner.Name;
        }
    }
}
```
Camille version 1.1.0 installed from NuGet.
.Net version: 4.6.1

**Comment by MingweiSamuel** [comment]
Fixed by 075c916aeab6261dc45dec634f1cc20c1e6d1ba1
